com.rustdesk.RustDesk.json

```diff
@@ -0,0 +1,91 @@
+            "url": "https://github.com/linux-pam/linux-pam/releases/download/v1.3.1/Linux-PAM-1.3.1.tar.xz",
+            "sha256": "eff47a4ecd833fbf18de9686632a70ee8d0794b79aecb217ebd0ce11db4cd0db"
+          }
+        ]
+      },
+      {
+        "name": "libarchive",
+        "buildsystem": "autotools",
+        "config-opts": ["--disable-static"],
+        "cleanup": ["/include", "/lib/*.la", "/lib/pkgconfig", "/share"],
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://github.com/libarchive/libarchive/releases/download/v3.5.1/libarchive-3.5.1.tar.gz",
+                "sha256": "9015d109ec00bb9ae1a384b172bf2fc1dff41e2c66e5a9eeddf933af9db37f5a"
+            }
+        ]
+      },
+      {
+        "name": "rustdesk",
+        "buildsystem": "simple",
+        "build-commands": [
+          "/app/bin/bsdtar -zxvf rustdesk*.deb",
+          "rm -rf /app/bin",
+          "tar -xvf ./data.tar.xz",
+          "cp -r usr/*  /app/",
+          "mkdir -p /app/bin && ln -s /app/lib/rustdesk/rustdesk /app/bin/rustdesk",
+          "mv /app/share/applications/rustdesk.desktop /app/share/applications/com.rustdesk.RustDesk.desktop",
+          "mv /app/share/applications/rustdesk-link.desktop /app/share/applications/com.rustdesk.RustDesk-link.desktop",
+          "sed -i '/^Icon=/ c\\Icon=com.rustdesk.RustDesk' /app/share/applications/*.desktop",
+          "install -Dm644 usr/share/icons/hicolor/256x256/apps/rustdesk.png /app/share/icons/hicolor/256x256/apps/com.rustdesk.RustDesk.png"
+        ],
+        "cleanup": ["/include", "/lib/pkgconfig", "/share/gtk-doc", "/share/man"],
+        "sources": [
+          {
+            "type": "file",
+            "only-arches": ["aarch64"],
+            "url": "https://github.com/rustdesk/rustdesk/releases/download/1.2.4/rustdesk-1.2.4-aarch64.deb",
+            "sha256": "4f8be7199a1e8acf0799ce8cea6af08bcf2d6f073a3a098809ac842e3a87d1e8",
+            "x-checker-data": {
+              "type": "anitya",
+              "project-id": "309593",
+              "url-template": "https://github.com/rustdesk/rustdesk/releases/download/$version/rustdesk-$version-aarch64.deb"
+            }
+          },
+          {
+            "type": "file",
+            "only-arches": ["x86_64"],
+            "url": "https://github.com/rustdesk/rustdesk/releases/download/1.2.4/rustdesk-1.2.4-x86_64.deb",
+            "sha256": "43c2f43ad7a8a173fa51093944243f04f003f675e74812644780eee1468b3f9b",
+            "x-checker-data": {
+              "type": "anitya",
+              "project-id": "309593",
+              "url-template": "https://github.com/rustdesk/rustdesk/releases/download/$version/rustdesk-$version-x86_64.deb"
+            }
+          }
+        ]
+      }
+    ],
+    "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--share=network",
+      "--filesystem=home",
```

Comment:
They are Remote Desktop software, transfer files is needed for end users

Reply:
File transfer

Reply:
Does it support portals? Do you need access to a specific directory?

Reply:
Why does AnyDesk flathub have home? But we can not have

Reply:
If you continue to insist on double standards, we may have to give up my submission.

Reply:
I didn't say you can't have. I'm asking to know if it is really needed.

Also what other applications do is irrelevant here. This PR is for this app.

Reply:
I do not know it is really needed, I am not flatpak expert.

Reply:
> I do not know it is really needed, I am not flatpak expert. You should not expect every one is flatpak expert.

But you do know what your application uses and what it doesn't? This does not require you to be a flatpak expert.

Reply:
Below two remote desktop can have home for the convience of users, but we can not have home because of your preference?
https://github.com/flathub/net.christianbeier.MultiVNC/blob/master/net.christianbeier.MultiVNC.yml#L12
https://github.com/flathub/com.anydesk.Anydesk/blob/635ebbbdb4185c5edeba387dad864a0612c54976/com.anydesk.Anydesk.json#L18

Reply:
This does not answer anything... Each application is separate just because someone else uses it doesn't mean you have to as well.

Reply:
Ok, I give up. Thanks for your time.

Reply:
> This does not answer anything... Each application is separate just because someone else uses it doesn't mean you have to as well.

I do not think so, you answer nothing either.

Reply:
It is ridiculous that the previous reviewer agree with the home permission of remote desktop software, now a new reviewer does not agree with it, though all of you are volunteers.

Reply:
what's ridiculous is that

1. you think you know better 
2. it's not about what the app does but how it is written
3. until recently filesystem permissions were not gated so they could have been changed
4. talking about double standard, you are making one right now by thinking you know better while not even being able to justify the choices made when asked.

Locking this thread now since the PR is closed.